### ISSUE TITLE
*Mental Self-care Toolbox* -- [Looking for Web Developers for Summer MVP]

### ISSUE THREAD

**Issue by MCharming98** [fix-the-internet/mozilla-builders#94]
** Project Name **
(we haven't come up with an official name yet)
Self-care/Positive Psychology Exercises Toolbox & Community 

** What you're looking for **
We are looking for teammates with web front/backend experiences, ideally in the US to accommodate the time difference & communication. 

** Short Project Description **
We aim to build a web application that offers people quick, accessible, and evidence-based self-relief exercises to address people’s daily emotional turbulence (e.g. peer pressure, conflicts in work, self-doubt…). The website will help individuals labeling their emotions, automatically generate a personalized ~5min guided self-care exercise for them, and lead them to see they are not alone by ultimately create an anonymous community along this theme. 

** Team Bios **
Our team currently has 4 rising seniors from UC Berkeley and UC San Diego (our backgrounds: 1 engineer with both front and backend skills, 1 product manager, 1 UI/UX designer, and 1 psychology and public health consultant/content producers). 

** Why Work With Us **
We are here to grow, learn and realize what we are passionate about :D Most importantly, we’d love to work with people who have interests in mental healthcare and human happiness in general:) If that sounds like you, definitely reach out to us!

Chen Meng - Tech Lead: https://www.linkedin.com/in/chenmeng98/

**Comment by ghost** [comment]
Hi @MCharming98, this looks great! Is that US-based requirement a hard one?

**Comment by MCharming98** [comment]
Hi @filipecatraia , it is not! Could you briefly describe the skills that you are good at?

**Comment by SinhaVinita** [comment]
Hi @MCharming98 I'm really impressed by the concept and the idea of your project- it looks great! I would love to get on board and contribute impactfully. I'm a rising junior at Cornell University, majoring in CS. I have industry and project experience with backend, full-stack development. I'd be happy to share more about my skillset!

**Comment by MCharming98** [comment]
Hi @SinhaVinita, can you briefly talk about your experiences? What languages and frameworks are you familiar with in terms of web dev? Also, feel free to to reach out to me on LinkedIn!

**Comment by SinhaVinita** [comment]
@MCharming98 I primarily work in Java, C++, and Python and have considerable experience with Javascript, C#, Go as well. As a software engineering intern at Magic Leap in Summer 2019, I interned on the Enterprise and Cloud team. My project as a full-stack developer was to develop a proof-of-concept application for data visualization and remote assistance to onsite engineers, using C++ on the Unreal Engine and Unity 3D for AR/VR headsets. I've worked with Python, Flask, and Heroku to build a web app at HackMIT 2019 and I recently worked on another web app using ReactJS and Firebase.
